src/render_loop_mlx.c

```diff
@@ -0,0 +1,35 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        :::      ::::::::   */
+/*   render_loop_mlx.c                                  :+:      :+:    :+:   */
+/*                                                    +:+ +:+         +:+     */
+/*   By: gwolf <gwolf@student.42vienna.com>         +#+  +:+       +#+        */
+/*                                                +#+#+#+#+#+   +#+           */
+/*   Created: 2023/12/19 22:37:57 by gwolf             #+#    #+#             */
+/*   Updated: 2023/12/19 22:54:05 by gwolf            ###   ########.fr       */
+/*                                                                            */
+/* ************************************************************************** */
+
+#include "render.h"
+
+int	ft_draw_scene(t_render *render)
```

Comment:
Do we need an extra function for that?

Reply:
Placeholder for now. Plan is to also render a menu (for example a bar on the left side) with info about the current object which would also be called here. 
Although the names should be flipped, since scene is the thing which gets raytraced.

Reply:
I see